flink-yarn/src/main/java/org/apache/flink/yarn/YarnApplicationFileUploader.java

```diff
@@ -220,7 +246,9 @@ YarnLocalResourceDescriptor registerSingleLocalResource(
 	 */
 	List<String> registerMultipleLocalResources(
 			final Collection<Path> shipFiles,
-			final String localResourcesDirectory) throws IOException {
+			final String localResourcesDirectory,
+			LocalResourceType resourceType
```

Comment:
Add `final` to the parameter (as done above) and also the closing `) throws IOException {` can be on the same line as the last parameter.